README.md

```diff
@@ -1,2 +1,172 @@
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+1-Web Browser
+2-Code Editor
+3-Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this Command:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Errytagedesign/mathMagician
+```
+
+### Install
+
+First you need to install the node package files
+Run this command:
+
+```sh
+  cd mathMagician
+  npm install
+```
+
+### Usage
+
+- In order to use this project you either create branch to add more features.
+- But in order to run the project run npm start
+- Run npm run build when you're done.
+
+### Run Linters
+
+I have added the linters or tests in order to avoid anykind of typos and ensure the good practices in this project. So while using this repository if you need to check your work in html and css you can use below commands
+
+- in order to run the linter test on index.html:
+
+```sh
+  npx eslint "**/*.{js,jsx}"
+```
+
+- in order to run the linter test on stylesheets:
+
+```sh
+ npx styleint "**/*.{css,scss}"
+```
+
+### Deployment
+
+- You can easily use github pages, netlify or vercel to deploy your project
+  <a href="https://docs.github.com/en/pages/quickstart">Useful link about Github Pages</a>
+  <a href="https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/">Step by Step guide to deploy using netlify</a>
+  <a href="https://www.programonaut.com/host-your-application-for-free-with-vercel-step-by-step/">Step by Step guide to deploy using vercel</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Sodeeq Awoyemi
+
+- GitHub: [@githubhandle](https://github.com/Errytagedesign)
+- Twitter: [@twitterhandle](https://twitter.com/errytage)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/errytagedesigns/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Create components dir
+- [ ] Create file into the components dir
+- [ ] Create math Magician projects
+- [ ] Hit The API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project just give it a star -
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] This link is broken. kindly consider fixing this as having broken links in our readme makes it unprofessional.